docs/aws/README.md

```diff
@@ -1,11 +1,40 @@
-# AWS
+# Getting Started with Karpenter on AWS
 
-This guide will provide a complete Karpenter installation for AWS.
-These steps are opinionated and may need to be adapted for your use case.
+Karpenter automatically provisions new nodes in response to unschedulable
+pods. Karpenter does this by observing events within the Kubernetes cluster,
+and then sending commands to the underlying cloud provider. 
 
-> This guide should take less than 1 hour to complete and cost less than $.25
+In this example, the cluster is running on Amazon Web Services (AWS) Elastic
+Kubernetes Service (EKS). Karpenter is designed to be cloud provider agnostic,
+but currently only supports AWS. Contributions are welcomed. 
+
+This guide should take less than 1 hour to complete, and cost less than $0.25.
```

Comment:
Can we change 1 hour -> 30 minutes. I don't want to scare folks off and it just doesn't take that long.

Reply:
I started a new EKS cluster build and if it takes less than 20 minutes I'll update this.

Reply:
it took only 11! 🥳 

it does *feel* like ages